File: zoo-back/Enitities/Empleado.cs
using System;
using System.Collections.Generic;

namespace Entities;

public partial class Empleado
{
    public int IdEmpleado { get; set; }

    public string Nombre { get; set; } = null!;

    public string Apellido1 { get; set; } = null!;

    public string Apellido2 { get; set; } = null!;

    public string Correo { get; set; } = null!;

    public int IdPuesto { get; set; }

    public int IdZoo { get; set; }

    public virtual Puesto IdPuestoNavigation { get; set; } = null!;

    public virtual Zoo IdZooNavigation { get; set; } = null!;

    public virtual ICollection<Tarea> Tareas { get; set; } = new List<Tarea>();

    public virtual Usuario? Usuario { get; set; }

    public virtual ICollection<VentaEntradum> VentaEntrada { get; set; } = new List<VentaEntradum>();
}

File: zoo-front/src/Types/next-auth-types/Empleado.ts
export interface RegistarEmpleado {
  idEmpleado: string;
  nombre: string;
  apellido1: string;
  apellido2: string;
  correo: string;
  idPuesto: number;
  contraseña: string;
  idRol: number;
  rolFechaInicio: string;
  rolFechaFin: string;
}

export interface Empleado {
  idEmpleado: string;
  nombre: string;
  apellido1: string;
  apellido2: string;
  puesto: string;
  roles: string
}